### ISSUE TITLE
ask to translate into indonesian language

### ISSUE THREAD

**Issue by mohdhaekal** [kennethreitz/python-guide#873]
hello sir ... may i translate into indonesian language...???
 for Indonesian people to read it easier.
thank you for your attention

**Comment by kennethreitz** [comment]
sure! just fork the repo and start working :)

**Comment by mohdhaekal** [comment]
yes I am very sure .. if you can ask from where I used to be translated

**Comment by dbader** [comment]
@mohdhaekal I suggest you start with the index page to lay out a skeleton with all of the missing content. For missing pages you can add links back to the English version at https://docs.python-guide.org/. That way you can give contributors a way to add Indonesian translations step by step.
